### DISCUSSION TITLE
Write python code for Upper_case alphabets pattern using for and while loop

### DISCUSSION THREAD

**Discussion by SaikumarS2611** [SaikumarS2611/Pattern#5] [Q&A]
Write a python codes

**Comment by saribalarakeshreddy** [comment]
okay I will sort it out  
 I am going to create a subrepo/subfolder named Alphabets and on that again a subrepo named Capital_Alphabets and upload the codes of them how about it??? if you agree with the names of subrepos are I will close this issue

**Comment by SaikumarS2611** [comment]
Write documentation for your Task

**Reply by saribalarakeshreddy** [reply]
okay I will and clarify me on on thing that how  and where I have to  create it and  in which format it has to be ?

**Reply by SaikumarS2611** [reply]
what I'm saying is write short documentation for your task 'Upper_case' alphabet letters pattern on your Alp folder
